Assets/MixedRealityToolkit.Tests/PlayModeTests/ObjectManipulatorTests.cs

```diff
@@ -662,18 +716,27 @@ private void RecordTransform(Transform transform, string description)
         public IEnumerator ObjectManipulatorOriginOffset()
         {
             TestUtilities.PlayspaceToOriginLookingForward();
+            // Without this background object that contains a collider, Unity will rarely
+            // return no colliders hit for raycasts and sphere casts, even if a ray / sphere is 
+            // intersecting the collider. Causes tests to be unreliable.
+            // It seems to only happen when one collider is in the scene.
+            var backgroundObject = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            backgroundObject.transform.position = Vector3.forward * 10;
+            backgroundObject.transform.localScale = new Vector3(100, 100, 1);
+            var backgroundmaterial = new Material(StandardShaderUtility.MrtkStandardShader);
+            backgroundmaterial.color = Color.green;
+            backgroundObject.GetComponent<MeshRenderer>().material = backgroundmaterial;
```

Comment:
nit: didn't we have this issue in another test as well? maybe worth moving into a util method so we can easier remove this hack in case that issue gets fixed?

Reply:
Oh, it's actually the same test, but for `ObjectManipulator`. The other test that needs to do this is  `ManipulationHandlerOriginOffset`. I thought I had added this commit to my earlier PR into `ux_refactor`, but I must have made a mistake when reordering commits. 
This commit just ensures parity between `ManipulationHandler` and `ObjectManipulator` tests